src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -0,0 +1,110 @@
+package nextstep.subway.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import nextstep.subway.constant.ErrorCode;
+import nextstep.subway.utils.StringUtils;
+
+@Entity
+public class Line extends BaseEntity {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
```suggestion
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
```
어노테이션은 한줄에 한개씩 사용해보면 어떨까요?
한줄에 한개씩 사용하는편이 가독성이 더 좋을 것 같아요 :)